pkg/config/config.go

```diff
@@ -53,6 +56,17 @@ func getCloudControllerManagerFromImages(platform configv1.PlatformType, images
 		return images.CloudControllerManagerAWS
 	case configv1.OpenStackPlatformType:
 		return images.CloudControllerManagerOpenStack
+	case configv1.AzurePlatformType:
+		return images.CloudControllerManagerAzure
+	default:
+		return ""
+	}
+}
+
+func getCloudNodeManagerFromImages(platform configv1.PlatformType, images imagesReference) string {
```

Comment:
This method should be tested, in case we would add new providers following similar pattern.